tests/appsec/iast/source/test_cookie_value.py

```diff
@@ -23,6 +23,7 @@ def setup_source_reported(self):
         self.source_fixture.setup()
 
     @bug(context.weblog_variant == "jersey-grizzly2", reason="name field of source not set")
+    @bug(library="nodejs", reason="Not reported correctly")
```

Comment:
The test expects a source value but in nodejs the value is redacted so the assertion is never true
```
{
  "origin": "http.request.cookie.name",
  "name": "table",
  "pattern": "abcde",
  "redacted": true
}
```
Should the test handle redacted values?